.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,45 @@
+name: Installation or build bug report
+description: Report installation or build bugs
+labels: ['Type: Bug']
+body:
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
+      options:
+        - label: I have read the documentation for [esp-protocols components](https://espressif.github.io/esp-protocols/) and the issue is not addressed there.
+          required: true
+        - label: I have updated my esp-protocols branch (master or release) to the latest version and checked that the issue is present there.
+          required: true
+        - label: I have searched the issue tracker for a similar issue and not found a similar issue.
+          required: true
+  - type: dropdown
+    id: component
+    attributes:
```

Could we also ask for the actual component version?

Something like?
```suggestion
    attributes:
      label: component version
      description: Which component version does this issue occur on? (see your `idf_component.yml`)
      placeholder: 1.0.0
```

---

Done